script/publish_persistent_udfs

```diff
@@ -71,7 +71,7 @@ def publish_persistent_udf(raw_udf, dataset, project_id):
             "CREATE TEMP FUNCTION", "CREATE OR REPLACE FUNCTION"
         )
 
-        client.query(query)
+        client.query(query).result()
```

Comment:
turns out `client.query` returns the job, which runs asynchronously and we need to check the result